pages/login/index.tsx

```diff
@@ -0,0 +1,89 @@
+import { Button, TextField } from "@mui/material";
+import Link from "next/link";
+import styles from "./styles.module.css";
+import { FormEvent, useState } from "react";
+import { FieldValues, useForm } from "react-hook-form";
+const Login = () => {
+  const {
+    register,
+    reset,
+    formState: { errors, isSubmitting },
+    handleSubmit,
+  } = useForm();
+
+  const emailErr = errors.email != undefined;
+  const passwordErr = errors.password != undefined;
+
+  const customeSubmit = (data: FieldValues) => {
+    console.log(data);
+
+    // validate with the server
+
+    // reset the form
+    reset();
+  };
+  return (
+    <div className={styles.formContainer}>
+      <div style={{ textAlign: "center", fontSize: "1.5rem" }}>
+        <h2>
+          <span style={{ color: "#263397" }}>s</span>
+          <span style={{ color: "#d3513e" }}>k</span>
+          <span style={{ color: "#e0b467" }}>o</span>
+          <span style={{ color: "#6bb7ee " }}>o</span>
+          <span style={{ color: "#c45946" }}>l</span>
+        </h2>
```

Comment:
You can use the Skool Icon in the public file instead of styling each character individually, but what you have is great also.